fe/src/pages/About/About.tsx

```diff
@@ -1,9 +1,11 @@
+import { Box } from '@chakra-ui/react';
+
 function About(): JSX.Element {
   // @TODO - Add some actual content here.
   return (
-    <div>
+    <Box h="100vh">
       <h1>About Page</h1>
-    </div>
+    </Box>
```

Comment:
Temporary visual change for "empty" pages, just to make current footer position lower.